src/test/java/plugins/GitPluginTest.java

```diff
@@ -209,6 +210,142 @@ public void update_submodules_recursively() throws IOException, InterruptedExcep
         job.startBuild().shouldSucceed();
     }
 
+    @Test
+    public void calculate_changelog() throws IOException, InterruptedException, SftpException, JSchException {
+        final URL changesUrl;
+        final String TEST_COMMIT_MESSAGE = "Second commit";
+        Build b;
+        GitRepo repo = buildGitRepo();
+
+        repo.branch("testBranch");
+        repo.commit(TEST_COMMIT_MESSAGE);
```

Comment:
Test fails here since there is nothing to commit. The branch in line 220 is already committed.